src/core/client/framework/lib/errors/unknownServerError.ts

```diff
@@ -21,6 +22,17 @@ export default class UnknownServerError extends Error {
       Error.captureStackTrace(this, UnknownServerError);
     }
 
+    const traceID: string | null | undefined = (error as any).traceID as string;
```

Comment:
Instead of casting `error as any` add `traceID` to  type `UnknownErrorExtension` on line 4.